pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -675,6 +675,14 @@ func (dm *DockerManager) runContainer(
 			// TODO: Clean up the previously created dir? return the error?
 			glog.Errorf("Error on creating termination-log file %q: %v", containerLogPath, err)
 		} else {
+			// Termination log should be writable by the process group. This is
+			// especially important in the remap environment because otherwise
+			// it won't be possible to write the status. Docker could adjust
+			// permissions on this file for us but it means that right after
+			// container creation, file will be non-writable for a short period.
+			if err := fs.Chmod(0664); err != nil {
```

Comment:
> It seems like there is a possibility of it chowing something on an already started container now. Ie. should it only be called in the ContainersToStart loop or possibly one of the called methods there like tryStartContainer?

Hm, looks like you're right and it's being called during the container lifetime:
```
I0124 17:05:26.670389   13894 docker_manager.go:2098] DEBUG: enter SyncPod()
I0124 17:05:28.693176   13894 docker_manager.go:1705] DEBUG: enter runContainerInPod()
I0124 17:05:28.694279   13894 docker_manager.go:586] DEBUG: enter runContainer()
I0124 17:05:29.525071   13894 docker_manager.go:2345] DEBUG: ensuring directory ownership for pod 4d012175-e257-11e6-bb4b-080027242396 is set to FSGroup 427680
I0124 17:05:29.525524   13894 docker_manager.go:2360] DEBUG: enter tryContainerStart()
I0124 17:05:31.629214   13894 docker_manager.go:1705] DEBUG: enter runContainerInPod()
I0124 17:05:31.629686   13894 docker_manager.go:586] DEBUG: enter runContainer()
I0124 17:05:31.631152   13894 docker_manager.go:656] DEBUG: creating termination-log file
I0124 17:05:33.255041   13894 docker_manager.go:2098] DEBUG: enter SyncPod()
I0124 17:05:33.255175   13894 docker_manager.go:2345] DEBUG: ensuring directory ownership for pod 4d012175-e257-11e6-bb4b-080027242396 is set to FSGroup 427680
I0124 17:05:34.258666   13894 docker_manager.go:2098] DEBUG: enter SyncPod()
I0124 17:05:34.258898   13894 docker_manager.go:2345] DEBUG: ensuring directory ownership for pod 4d012175-e257-11e6-bb4b-080027242396 is set to FSGroup 427680
I0124 17:05:38.264070   13894 docker_manager.go:2098] DEBUG: enter SyncPod()
I0124 17:05:38.502884   13894 docker_manager.go:2345] DEBUG: ensuring directory ownership for pod 4d012175-e257-11e6-bb4b-080027242396 is set to FSGroup 427680
```